apps/payments/next/app/[locale]/[offeringId]/checkout/[interval]/[cartId]/success/en.ftl

```diff
@@ -1,8 +1,8 @@
-next-payment-confirmation-thanks-heading = Thank you!
+next-payment-confirmation-thanks-heading = Thanks, now check your email!
```

Comment:
We can re-use the string below (just add the `next-` prefix).
```
payment-confirmation-thanks-heading-account-exists = Thanks, now check your email!
```

Reply:
I'd be happy to update it! Does re-using the string with an added `next-` prefix help improve translations?

Reply:
Adding the `next-` prefix helps us keep track of which strings already have translations. :)

Reply:
Just a bit of additional context to what Lisa mentioned. The intention is to reuse translations from SP2.0 in SP3.0, i.e. copy strings from `payments.ftl` into `payments-next.ftl`. The way the reusable strings are currently identified is by prefixing the id's with `next-`.